include/bx/handlealloc.h

```diff
@@ -79,7 +79,8 @@ namespace bx
 		~HandleAllocT();
 
 	private:
-		uint16_t m_padding[2*MaxHandlesT];
```

Comment:
Do not change code when adding test...

This is intentional single buffer since code analyzers complain when they see that you're accessing out of bounds array.